ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/page/v1/page/README.md

```diff
@@ -0,0 +1,30 @@
+
+<!--
+Copyright 2024 Adobe
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+Adaptive Form Core Page (v1)
+====
+Adaptive Form Core Page component v1 which extends the wcm v3 [Page](https://github.com/adobe/aem-core-wcm-components/tree/d50ba1cb0d6544870daff71281bf9dfd3f6fbeb0/content/src/content/jcr_root/apps/core/wcm/components/page/v3/page) Component.
```

Comment:
you don't need to mention the specific version of WCM core component

Reply:
version removed